### ISSUE TITLE
Intelligence Office on Almayer is all access. 

### ISSUE THREAD

**Issue by troytroy400** [RMC-14/RMC-14#3972]
## Description
Every single door in the Intelligence Office on the second floor of the Almayer is all access.

**Reproduction**
Attempt to open an IO door with no ID equipped. 

**Screenshots**
N/A

**Additional context**
N/A

**Comment by Vermidia** [comment]
Was this fixed?

**Comment by Dutch-VanDerLinde** [comment]
> Was this fixed?

No, was able to access intel office as bravo comtech post intel update
